scripts/validate_solutions.py

```diff
@@ -5,7 +5,7 @@
 import os
 import pathlib
 from types import ModuleType
-
+import datetime
```

Comment:
Also you need to run` pre-commit `

- Install, before you can run hooks, you need to have the pre-commit package manager installed. >`pip install pre-commit`

then > `pre-commit`

- How to expect?

`run python pre-commit-#.#.#.pyz ... in place of pre-commit ...`